ftplugin/php/splitjoin.vim

```diff
@@ -1,6 +1,6 @@
 let b:splitjoin_split_callbacks = [
       \ 'sj#js#SplitArray',
-      \ 'sj#php#SplitArray',
+      \ 'sj#php#SplitBraces',
```

Comment:
There seems to be a build error, now, because this:

``` php
if ($foo) { $a = $bar; }
```

is supposed to split into:

``` php
if ($foo) {
  $a = $bar;
}
```

But now it splits into:

``` php
if (
  $foo
) { $a = $bar; }
```

I think it's easy to fix by just swapping the `SplitBraces` function with the `SplitIfClause` one, since the list is attempted in order.